website/src/layouts/OrganismPage/AccessionsDownloadButton.astro

```diff
@@ -0,0 +1,61 @@
+---
+import type { LapisFilter } from '@genspectrum/dashboard-components/util';
+
+import { assembleDownloadUrl } from './assembleDownloadUrl';
+import { getLapisUrl } from '../../config';
+import { Modal } from '../../styles/containers/Modal';
+import { ModalContent } from '../../styles/containers/ModalContent';
+import { ModalHeader } from '../../styles/containers/ModalHeader';
+import type { OrganismConstants } from '../../views/OrganismConstants';
+import type { View } from '../../views/View';
+import type { PageStateHandler } from '../../views/pageStateHandlers/PageStateHandler';
+
+type DownloadLink = {
+    label: string;
+    filter: LapisFilter;
+    downloadFileBasename: string;
+};
+
+interface Props {
+    view: View<object, OrganismConstants, PageStateHandler<object>>;
+    downloadLinks: DownloadLink[];
+}
+
+const { view, downloadLinks } = Astro.props;
+
+const modalId = 'accessionDownloadModal';
+---
+
+<button class='mr-4 flex items-center p-2 text-xs' onclick={`${modalId}.showModal()`}>
+    <span class='iconify text-base mdi--download'></span>
+    Download data
+</button>
+<Modal id={modalId}>
+    <ModalHeader title='Download data' icon='mdi--download' />
+    <ModalContent>
+        <p>You can download a list of accessions that are used for the visualizations on this page:</p>
+        <div class='m-8'>
+            {
+                downloadLinks
+                    .map(({ label, filter, downloadFileBasename }) => ({
+                        label,
+                        url: assembleDownloadUrl(
+                            view.organismConstants.accessionDownloadFields,
+                            filter,
+                            downloadFileBasename,
+                            getLapisUrl(view.organismConstants.organism),
+                        ),
+                    }))
+                    .map(({ label, url }) => (
+                        <a class='link m-2 flex items-center' href={url}>
+                            <span class='iconify mdi--download' />
```

Comment:
```suggestion
                        <a class='link m-2 flex items-center gap-1' href={url}>
                            <span class='iconify mdi--download text-xl' />
```
I would prefer, that the button has the same size as the capital "D" of "Download", and also align the underline with the underline of the link.

Reply:
It's still not properly aligned, but better.